atlas-search/public-api/find.md

```diff
@@ -290,102 +293,74 @@ query FindQuery(
+```json
 {
   "query": "Austin",
   "filter": "post_type:post,page",
   "orderBy": [
-    { "field": "published_at", "direction": "desc" }
+    { "field": "published_at", "direction": "desc", "unmappedType": "date" }
   ],
   "offset": 200,
   "limit": 100,
-  "fields": [
-    { "name": "title", "weight": 2 },
-    { "name": "description" }
-  ],
+  "fields": [{ "name": "title", "weight": 2 }, { "name": "description" }],
   "tolerance": { "name": "fuzzy", "fuzzyDistance": 2 }
 }
 ```
 
-<table>
-  <thead>
-    <tr>
-      <th>Parameter</th>
-      <th>Description</th>
-    </tr>
-  </thead>
-  <tbody>
-    <tr>
-      <td>query</td>
-      <td>The search query itself. We search for documents containing the word 'Austin'.</td>
-    </tr>
-    <tr>
-      <td>filter</td>
-      <td>Additional filtering to apply to the search results. We search only for posts and page post_types.</td>
-    </tr>
-    <tr>
-      <td>orderBy</td>
-      <td>Order by name asc and, in case of a tie, date desc.</td>
-    </tr>
-    <tr>
-      <td>offset</td>
-      <td>The number of results to skip for pagination. We asked to skip the first 200.</td>
-    </tr>
-    <tr>
-      <td>limit</td>
-      <td>The maximum number of results to return. We asked for 100.</td>
-    </tr>
-    <tr>
-      <td>fields</td>
-      <td>An array of objects specifying the fields to search for in the documents and their weights. Title has double the weight of description, so it will come first as a result.</td>
-    </tr>
-    <tr>
-      <td>tolerance</td>
-      <td>The search tolerance option, such as fuzzy search with a specified distance. We used fuzzy with distance 2.</td>
-    </tr>
-  </tbody>
-</table>
+The query returns a `SearchResult` object that includes fields:
 
-### Using Query Operators
+- `total` - an integer of the total documents matching the query. Independent to offset and limit arguments.
+- `documents` - and array of search documents with fields:
+  - `id` is unique id of the document in the index
+  - `score` is a relevance score, which determines how closely each document matches the query
+  - `sort` is an array associated with a document. The first element represents the sorting score or relevance score of the document. The second element represents a timestamp or any other value used for sorting.
 
-- Using `NOT` search operator:
+## Advanced Usage
+
+### Logical operators: AND, OR, NOT.
+
+Using `NOT` search operator:
 
 ```graphql
 {
   "query": "Austin NOT Minnesota",
 }
 ```
 
-- Using `AND` search operator:
+Using `AND` search operator:
 
 ```graphql
 {
   "query": "New York AND Texas",
 }
 ```
 
-- Using `OR` search operator:
+Using `OR` search operator:
 
 ```graphql
 {
   "query": "New York OR Texas",
 }
 ```
 
-### Advanced Query Example
+### Search field value with greater, lower or equal operator
 
 ```graphql
 {
-  "query": "seats.count::>4 !seats.count:10 tags.name:Cars hello AND world OR 123 4.56",
+  "query": "seats.count:(>4 AND <20)"
```

Comment:
I'm thinking we should keep graphql examples for these without variables as they are easier to copy and paste
```graphql
query QueryGreaterLowerOperators {
  find(query: "(seats.count:(>4 AND <20)") {
    total
    documents {
      id
      score
      data
      score
    }
  }
}
```

Reply:
if you are using postman or any other interface to send queries they have query area and variables area as well